run.sh

```diff
@@ -1,7 +1,10 @@
 #!/usr/bin/env bash
 
+
```

Comment:
Whitespaces should not pollute the PR diff. Make sure this file does not get listed .